apps/mobile-wallet/src/screens/SwitchNetworkModal.tsx

```diff
@@ -45,7 +47,6 @@ const SwitchNetworkModal = ({ onClose, onCustomNetworkPress, ...props }: SwitchN
     setSelectedNetworkName(newNetworkName)
 
     if (newNetworkName === NetworkNames.custom) {
-      onClose && onClose()
```

Comment:
Done

Reply:
Heyman, how do you get passed this so that I can open the settings modal to see if the above issue is fixed?

<img width="559" alt="image" src="https://github.com/user-attachments/assets/eb195821-23a1-40ee-899b-3a4f2075170e">